crates/typst-docs/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use std::{
+    fs::{create_dir_all, write},
+    path::{Path, PathBuf},
+};
```

Comment:
```suggestion
use std::fs;
use std::path::{Path, PathBuf};
```
and then `fs::create_dir_all` and `fs::write` (in particular the latter is easier to understand that way in my opinion).